content/en/admin/metrics.md

```diff
@@ -0,0 +1,86 @@
+| Timing | `cache.generate.duration`  |             |
+| Timing | `cache.read_hit.duration`  |             |
+| Timing | `cache.read_miss.duration` |             |
+| Timing | `cache.read_miss.duration` |             |
+
+### Sidekiq:
+
+| Type      | Name                                    | Description |
+| --------- | --------------------------------------- | ----------- |
+| Timing    | `sidekiq.{worker_name}.processing_time` |             |
+| Increment | `sidekiq.{worker_name}.success`         |             |
+| Increment | `sidekiq.{worker_name}.failure`         |             |
+| Gauge     | `sidekiq.queues.{queue_name}.enqueued`  |             |
+| Gauge     | `sidekiq.queues.{queue_name}.latency`   |             |
+| Gauge     | `sidekiq.dead_size`                     |             |
+| Gauge     | `sidekiq.enqueued`                      |             |
+| Gauge     | `sidekiq.failed`                        |             |
+| Gauge     | `sidekiq.processed`                     |             |
+| Gauge     | `sidekiq.processes_size`                |             |
+| Gauge     | `sidekiq.retry_size`                    |             |
+| Gauge     | `sidekiq.scheduled_size`                |             |
+| Gauge     | `sidekiq.workers_size`                  |             |
+
+## Streaming
+
+The streaming server reports metrics via a prometheus metrics endpoint located
+at `/metrics` on the streaming server. In order to collect these metrics, you'll
+need to make a request directly to the streaming server.
+
+The following metrics are available:
+
+| Type      | Name                      | Description                                                                                                                    |
+| --------- | ------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| Gauge     | pg_pool_total_connections | The total number of clients existing within the postgresql pool                                                                |
+| Gauge     | pg_pool_idle_connections  | The number of clients which are not checked out but are currently idle in the postgresql pool                                  |
+| Gauge     | pg_pool_waiting_queries   | The number of queued requests waiting on a postgresql client when all clients are checked out                                  |
+| Gauge     | connected_clients         | The number of clients connected to the streaming server, these are labelled with the type of connection: `http` or `websocket` |
+| Gauge     | connected_channels        | The number of Redis channels the streaming server is subscribed to                                                             |
+| Histogram | streaming_latency         | The time delta between when an event is published to redis from Ruby and when the streaming server processes that event        |
```

Comment:
Note: this is a new metric (see: https://github.com/mastodon/mastodon/pull/24886/ )